openskistats/models.py

```diff
@@ -252,6 +252,11 @@ class SkiAreaModel(Model):  # type: ignore [misc]
     hemisphere: Literal["north", "south"] | None = Field(
         description="Hemisphere of the ski area.",
     )
+    solar_intensity: float | None = Field(
+        description="Solar intensity of the ski resort calculated by the declination at that latitude at the solstice",
```

Comment:
Specify which solstice.

What does solar intensity mean? Is it insolation scaled in some way? E.g. how does one interpret solar_intensity of 60%?